src/sessionOne/BasicTypes.kt

```diff
@@ -1,2 +1,168 @@
+    println("Age : $myAge")
+    println("Work : $work")
+    println("Religion : $religion")
+    println("ID Number : $idNumber")
+
+    // 2) print your ID after 2 years
+    println("Name : $myName")
+    println("Age : ${myAge + 2}")
+    println("Work : $work")
+    println("Religion : $religion")
+    println("ID Number : $idNumber")
+
+    // Basic Operations
+
+    // 1) a pointer (mouse) and a screen size, showing addition/subtraction operations
+    var x = 20
+    var y = 30
+    // screen size is constant
+    val length = 50
+    val width = 100
+    // create a string (text)
+    val display = "Screen Dimensions = L($length) x W($width)."
+    println(display)
+    println("Pointer location = (x($x), y($y))")
+    // move the pointer upward on the y-axis
+    y = y + 10
+    // move the pointer horizontally on the x-axis
+    x = x + 10
+    // print the new location
+    println("Pointer location = (x($x), y($y))")
+    // move the pointer back to its origin
+    y = y - 10
+    x = x - 10
+    // print the new location
+    println("Pointer location = (x($x), y($y))")
+    // print the current screen state in a different format
+    //  \n is an escape sequence, means new line
+    // the plus sign can concatenate strings or text together
+    // println() vs print(), println() prints the message in a new line while print() prints the message with
+    // the same line as previous message do.
+    print("----------------------\n"
+            +"| Retina Screen : | \n" +
+            "| $display | \n" +
+            "| Pointer location = (x($x), y($y)) | \n"
+            +"----------------------\n")
+
+    // equivalent operations
+    // 'x += 10' is the same as 'x = x + 10'
+    // '++x' is called a prefix increment the same as 'x += 1' and 'x = x + 1'
+    // 'x++' is called a postfix increment, its same as :
+    // 'x = x' at this line
+    // 'x = x + 1' at the next line
+
+    // Boolean operations for dummies -- Boolean Algebra --
+    // think of logic gates when it comes to booleans or the probability of certainty/truth and impossibility/lie (0 or 1)
+    // true means 1 means certain
+    // false means 0 means impossible
+    var probability1 = true
+    var probability0 = false
+    // when 1 (true) and 0 (false) are combined in a logic gate AND the result would be 0 (false)
+    // the AND gate is designated by && symbol for logical AND
+    println("Test booleans AND : ${probability0 && probability1}")
+    // when 1 and 0 are combine in a logic gate OR the result would be 1
+    // the OR gate is designated by || symbol for logical OR
+    println("Test booleans OR : ${probability0 || probability1}")
+    // when 1 (true) is inverted, the result is zero (false)
+    // ! is called the inverter gate or the logical inverse
+    println(!probability1)
+    // when 0 (false) is inverted, the result is one (true)
+    // ! is called the inverter gate or the logical inverse
+    println(!probability0)
+
```

Comment:
- Casting
- Exception Handling 
are in progress !

Reply:
Control flow: if, else if, else , when